OSDOCS-14588: Updates to egress lockdown documentation (#93240)
openshift/openshift-docs: modules/rosa-hcp-create-network.adoc

INFO[0140] Stack rosa-network-stack-5555 created \ <2>
----
<1> These two subnet IDs are used to create your cluster when using the `rosa create cluster` command.
<2> The network stack name is used to delete the resource later.

ifdef::rosa-disconnected-install[]
[discrete]
[id="rosa-hcp-vpc-subnet-tagging_{context}"]
== Tagging your subnets

> Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly before you can use these resources. The following table shows how your resources should be tagged as the following:

Reviewer: ```suggestion
Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly before you can use these resources. The following table shows how your resources should be tagged:
```

[cols="3a,8a,8a", options="header"]
|===
| Resource
| Key
| Value

| Public subnet
| `kubernetes.io/role/elb`	
| `1` or no value

| Private subnet 
| `kubernetes.io/role/internal-elb`	
| `1` or no value

|===

[NOTE]
====

> You must tag at least one private subnet and, if applicable, and one public subnet.

Reviewer: ```suggestion
You must tag at least one private subnet and one public subnet, if applicable.
```

====

. Tag your resources in your terminal by running the following commands:
.. For public subnets, run:
+
[source,terminal]
----
$ aws ec2 create-tags --resources <public-subnet-id> --region <aws_region> --tags Key=kubernetes.io/role/elb,Value=1
----
.. For private subnets, run:
+
[source,terminal]
----
$ aws ec2 create-tags --resources <private-subnet-id> --region <aws_region> --tags Key=kubernetes.io/role/internal-elb,Value=1
----

.Verification

> * Verify that the tag is correctly applied by running the following command:

Reviewer: ```suggestion
* Verify that the tag is correct by running the following command:
```

INFO[0140] Stack rosa-network-stack-5555 created \ <2>
----
<1> These two subnet IDs are used to create your cluster when using the `rosa create cluster` command.
<2> The network stack name is used to delete the resource later.

ifdef::rosa-egress-lockdown[]
[discrete]
[id="rosa-hcp-vpc-subnet-tagging-rosa-network_{context}"]
== Tagging your subnets

> Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly. The following table shows how your resources should be tagged:

Reviewer 2: ```suggestion
Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly. The following table shows how to tag your resources:
```

- Active voice + should is avoided

https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s

> Whenever possible, rewrite in a more direct way. Do not use should to refer to an action that must be performed.

[cols="3a,8a,8a", options="header"]
|===
| Resource
| Key
| Value

| Public subnet
| `kubernetes.io/role/elb`	
| `1` or no value

| Private subnet 
| `kubernetes.io/role/internal-elb`	
| `1` or no value

|===

[NOTE]
====
You must tag at least one private subnet and one public subnet, if applicable.
====

> . Tag your resources in your terminal by running the following commands:
> .. For public subnets, run:

Reviewer 2: ```suggestion
. Tag your resources in your terminal: 
.. For public subnets, run the following command:
```

- to be consistent with modules/rosa-hcp-vpc-manual.adoc, add "run the following command:"

+
[source,terminal]
----
$ aws ec2 create-tags --resources <public-subnet-id> --region <aws_region> --tags Key=kubernetes.io/role/elb,Value=1
----

> .. For private subnets, run:

Reviewer 2: ```suggestion
.. For private subnets, run the following command:
```

- to be consistent with modules/rosa-hcp-vpc-manual.adoc, add "run the following command:"

+
[source,terminal]
----
$ aws ec2 create-tags --resources <private-subnet-id> --region <aws_region> --tags Key=kubernetes.io/role/internal-elb,Value=1
----

.Verification

* Verify that the tag is correct by running the following command:
+
[source,terminal]
----
$ aws ec2 describe-tags --filters "Name=resource-id,Values=<subnet_id>"
----
+
.Example output
+
[source,text]
----

> TAGS    Name                    <subnet-id>        subnet  <prefix>-subnet-public1-us-east-1a
> TAGS    kubernetes.io/role/elb  <subnet-id>        subnet  1

Reviewer 2: ```suggestion
TAGS    Name                    <subnet_id>        subnet  <prefix>-subnet-public1-us-east-1a
TAGS    kubernetes.io/role/elb  <subnet_id>        subnet  1
```

Same comment as other module
